api/src/test/java/com/sanction/thunder/models/PilotUserTest.java

```diff
@@ -74,7 +88,15 @@ public void testHashCodeDifferent() {
 
   @Test
   public void testToString() {
-    String expected = "PilotUser [email=test@test.com]";
+    String email = new StringJoiner(", ", "Email [", "]")
+            .add("address=test@test.com")
+            .add("verified=true")
+            .add("verificationToken=hashToken")
+            .toString();
```

Comment:
If you test the Email's `toString()` in the `EmailTest` then you won't need to have this here.